src/server/models/user.model.js

```diff
@@ -6,10 +6,12 @@ var Schema = mongoose.Schema;
 // var ObjectId = Schema.ObjectId;
 
 var userSchema = new Schema({
-	// username: {
-	// 	type: String,
-	// 	index: true
-	// },
+	username: {
+		type: String,
+		index: true,
+		min: 4,
+		max: 20
+	},
```

Comment:
Joe says we should be doing both, so is that going to be a future commit or do you want to add it to this pull request?